legal-api/src/legal_api/reports/business_document.py

```diff
@@ -806,7 +806,27 @@ def _get_legal_type_description(legal_type: str) -> str:
         'CC': 'BC Community Contribution Company',
         'LLC': 'Limited Liability Company'
     }
+    
+    @staticmethod
+    def _format_role_type(role_type: str) -> str:
+        if not role_type:
+            return role_type
+        
+        role_display_mapping = {
+            'ceo': 'Chief Executive Officer',
+            'cfo': 'Chief Financial Officer',
```

Comment:
what about cto?!?!? "Chief Technical Officer" :P

Reply:
we currently don't support role cto (probably future) 
ref: https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/party_role.py#L31

Reply:
I was just kidding Vysakh